fog/view/server/src/fog_view_service.rs

```diff
@@ -177,15 +186,39 @@ where
         })
     }
 
+    /// Unwrap and forward to enclave
+    pub fn query_nonce_impl(
+        &mut self,
+        request: attest::NonceMessage,
+    ) -> Result<attest::NonceMessage, RpcStatus> {
+        log::trace!(self.logger, "Getting encrypted request");
+        let tracer = tracer!();
+
+        tracer.in_span("query_impl", |_cx| {
```

Comment:
:wrench: I know this has a TODO, but I'm guessing it will get missed in the update
```suggestion
        tracer.in_span("query_nonce_impl", |_cx| {
```

Reply:
Done.